### ISSUE TITLE
An error occurred: builder error

### ISSUE THREAD

**Issue by KoffeinKaio** [crunchy-labs/crunchy-cli#370]
**Describe the bug**
After a day or so the following error starts to pop up:
```
:: ✔ Logging in                                                                                                                                                                                                                                                                 :: ✔ Parsed url 1                                                                                                                                                                                                                                                               :: ✔ Loaded series information for url 1                                                                                                                                                                                                                                        :: SPY x FAMILY Season 1 (SPY x FAMILY)
::      1. OPERATION STRIX » S01E01
::      2. SECURE A WIFE » S01E02
::      3. PREPARE FOR THE INTERVIEW » S01E03
::      4. THE PRESTIGIOUS SCHOOL'S INTERVIEW » S01E04
::      5. WILL THEY PASS OR FAIL? » S01E05
::      6. THE FRIENDSHIP SCHEME » S01E06
::      7. THE TARGET'S SECOND SON » S01E07
::      8. THE COUNTER-SECRET POLICE COVER OPERATION » S01E08
::      9. SHOW OFF HOW IN LOVE YOU ARE » S01E09
::      10. THE GREAT DODGEBALL PLAN » S01E10
::      11. STELLA » S01E11
::      12. PENGUIN PARK » S01E12
::      13. PROJECT APPLE » S01E13
::      14. DISARM THE TIME BOMB » S01E14
::      15. A NEW FAMILY MEMBER » S01E15
::      16. YOR'S KITCHEN / THE INFORMANT'S GREAT ROMANCE PLAN » S01E16
::      17. CARRY OUT THE GRIFFIN PLAN / FULLMETAL LADY / OMELET RICE♥ » S01E17
::      18. UNCLE THE PRIVATE TUTOR / DAYBREAK » S01E18
::      19. A REVENGE PLOT AGAINST DESMOND / MAMA BECOMES THE WIND » S01E19
::      20. INVESTIGATE THE GENERAL HOSPITAL / DECIPHER THE PERPLEXING CODE » S01E20
::      21. NIGHTFALL / FIRST FIT OF JEALOUSY » S01E21
::      22. THE UNDERGROUND TENNIS TOURNAMENT: THE CAMPBELLDON » S01E22
::      23. THE UNWAVERING PATH » S01E23
::      24. THE ROLE OF A MOTHER AND WIFE / SHOPPING WITH FRIENDS » S01E24
::      25. FIRST CONTACT » S01E25
:: SPY x FAMILY Season 2 (SPY x FAMILY (Eps 26+))
::      1. FOLLOW MAMA AND PAPA » S02E26
::      2. BOND'S STRATEGY TO STAY ALIVE / DAMIAN'S FIELD RESEARCH TRIP » S02E27
::      3. MISSION AND FAMILY / THE ELEGANT BONDMAN / THE HEART OF A CHILD / WAKING UP » S02E28
::      4. The Pastry of Knowledge / The Informant's Great Romance Plan II » S02E29
::      5. Plan to Cross the Border » S02E30
::      6. The Fearsome Luxury Cruise Ship » S02E31
::      7. Who Is This Mission For? » S02E32
::      8. The Symphony Upon the Ship » S02E33
::      9. The Hand That Connects to the Future » S02E34
::      10. Enjoy the Resort to the Fullest / Bragging About Vacation » S02E35
::      11. Berlint in Love / Nightfall's Daily Life » S02E36
::      12. Part of the Family » S02E37
:: An error occurred: builder error

```


**To Reproduce**
Steps / command to reproduce the behavior:
```
$ crunchy-cli archive -y --skip-specials --skip-existing --skip-existing-method audio --skip-existing-method subtitle -a ja-JP -a de-DE -s en-US -s de-DE -o "./{series_name}/Season {season_number}/S{season_number}E{episode_number}.mkv" https://www.crunchyroll.com/de/series/G4PH0WXVJ/spy-x-family
```

**Expected behavior**
Expected to place the Episode in a format like:
```
./SPY x FAMILY/Season 02/S02E01.mkv
```

**Client (please complete the following information):**
 - OS: Archlinux
 - Version crunchy-cli 3.4.0 (18534b2 2024-04-08)

**Additional context**
Worked before, didnt update between it working and not - maybe some missing data from crunchyroll?
Maybe because I'm not premium anymore?

**Comment by bytedream** [comment]
Did you use a pre-built binary or built it from source?

**Comment by KoffeinKaio** [comment]
I built it from source

**Comment by bytedream** [comment]
Can you try again with a [pre-built](https://github.com/crunchy-labs/crunchy-cli/releases/tag/v3.4.0) binary (or the `crunchy-cli-bin` package from the AUR if you prefer the AUR) and report if the error still occurs?

**Comment by KoffeinKaio** [comment]
Sure thing:

```
$ crunchy-cli --version
crunchy-cli 3.4.0 (18534b2 2024-04-08)
$ pacman -Q crunchy-cli-bin
crunchy-cli-bin 3.4.0-1
```

Same error, works if I remove the "{series_name}" template

**Comment by Kartatz** [comment]
I was having this same issue yesterday with the prebuilt binary of the `v3.3.5` release.

> Same error, works if I remove the "{series_name}" template

This does not seem to be related to any formatting options. I was not using any of these when the error occurred.

 This happens randomly.

**Comment by KoffeinKaio** [comment]
> > Same error, works if I remove the "{series_name}" template
> 
> This does not seem to be related to any formatting options. I was not using any of these when the error occurred.
> 
> This happens randomly.

dunno, pretty reproducible with {series_name} and without

**Comment by bytedream** [comment]
This had something to do with you not using a premium account. Fixed in [v3.4.1](https://github.com/crunchy-labs/crunchy-cli/releases/tag/v3.4.1)

**Comment by kylobylo** [comment]
I still have this issue with v3.4.1
